update install document for user (#388)
erda-project/erda: docs/guides/deploy/How-to-install-the-Erda.md

	
	```shell
	tar -xzvf erda-release.tar.gz
	cd erda-release
	```
	
	​	

2. Apply Erda necessary configurations on the **Kubernetes Master Node**.

   - make sure the **kubeconfig** file on the ~/.kube/config.

   - set configruation to prepare the Erda and execute the `prepare.sh` script

     - The script will do the following tasks:

       - generate etc ssl
       
       - generate multi cluster manager ssl
       
       - set node labels which use for Erda Application
       
       - set Erda installer configuration   
       
         

     ```shell
     # specify the Kuberentes namespace to install erda components, default  value is default and the Erda components are only support default namespace.
     
     export ERDA_NAMESPACE="default"
     
     # specify the generic domains like *.erda-demo.erda.io to visit the erda application, default values is erda-demo.erda.io, you can set owner generic domains in here
     
     export ERDA_GENERIC_DOMAIN="erda-demo.erda.io"
     
     # Execute the script to apply Erda necessary configuration
     bash scripts/prepare.sh
     ```

   - update the values of `insecure-registries` 

> 

Reviewer: ```suggestion
   - update the values of `insecure-registries` in the docker daemon config and restart docker
```

   helm install package/erda-addons-0.1.0.tgz --generate-name
   
   # install erda
   helm install package/erda-0.1.0.tgz --generate-name
   ```

   

4. After Installed the Erda：

   - set admin username and password to push the Erda extensions（the extension is a plugin which uses in the pipeline）

     ```shell
     export ERDA_ADMIN_USERNAME=admin
     export ERDA_ADMIN_PASSWORD=password123456
     
     bash scripts/push-ext.sh
     ```

   - write the following URLs to `/etc/hosts` on the **machine where the browser is located**, replace the <IP> with IP of the **LB machine**.

     > For example, if I have a LB machine which IP is `10.0.0.1`, ERDA_GENERIC_DOMAIN is `erda-demo.erda.io`, org-name is `erda-test`. so I can write the following info to `/etc/hosts` 

     ```shell
     10.0.0.1 nexus.erda-demo.erda.io
     10.0.0.1 sonar.erda-demo.erda.io
     10.0.0.1 dice.erda-demo.erda.io
     10.0.0.1 uc-adaptor.erda-demo.erda.io
     10.0.0.1 soldier.erda-demo.erda.io
     10.0.0.1 gittar.erda-demo.erda.io
     10.0.0.1 collector.erda-demo.erda.io
     10.0.0.1 hepa.erda-demo.erda.io
     10.0.0.1 openapi.erda-demo.erda.io
     10.0.0.1 uc.erda-demo.erda.io

>      # example for <IP> <org-name>.<ERDA_GENERIC_DOMAIN>

Reviewer: ```suggestion
     # Note: The org name of this example is erda-test
```
